CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.6 (unreleased)
+
+### Summary
+
+- [#173](https://github.com/awslabs/amazon-s3-find-and-forget/pull/173): Show
```

Comment:
Since there's two changes (the interface change, and the additional types change), what about listing them as two points in the summary list.

Reply:
Ok done